BUILD

```
@@ -6,6 +6,8 @@ load(
     "swift_compiler_plugin"
 )
 
+swiftFeatures = ["-enable-upcoming-feature", "ExistentialAny"]
```

Nit: call this `copts` since in rules_swift, "features" are a different thing. E.g. https://github.com/bazelbuild/rules_swift/blob/1.12.0/swift/internal/feature_names.bzl

```suggestion
copts = ["-enable-upcoming-feature", "ExistentialAny"]
```